swanjala/musepadpocket
# Ch base project best practices

## What does this PR do?
- Refactors app with naming conventions
- Extracts values to values folder 

## How to test this?
- Check that the application properly displays all the values set
- Ensure that that constraint layout design is maintained
